locales/templates/app.pot

```diff
@@ -763,10 +763,18 @@ msgctxt "rate_app_dialog_btn_go_rate"
 msgid "Yes, rate 5 stars!"
 msgstr ""
 
+msgctxt "rate_app_notification_action_rate"
+msgid "RATE 5 STAR"
```

Comment:
Shouldn't this be "rate 5 starS"?

Reply:
Thanks! It should be 5 "stars".

Reply:
And also sorry for the deadline. It's my mistake.

Reply:
@cnevinc: Ok can you commit again to fix this? Then i'll be able to merge. Thanks!